src/main/app/paths.ts

```diff
@@ -10,6 +10,7 @@ export class Paths {
   static readonly cookiesPage = new RoutablePath('/cookies', false)
   static readonly termsAndConditionsPage = new RoutablePath('/terms-and-conditions', false)
   static readonly contactUsPage = new RoutablePath('/contact-us', false)
+  static readonly cmcCitizenFrontendMaintenancePage = new RoutablePath('/cmc-citizen-frontend-maintenance', false)
```

Comment:
please remove cmc-citizen-frontend from the path, should `-unplanned` be in the path?